internal/component/loki/process/process.go

```diff
@@ -147,7 +158,10 @@ func (c *Component) handleIn(ctx context.Context, wg *sync.WaitGroup) {
 			case <-ctx.Done():
 				return
 			case c.processIn <- entry.Clone():
-				// no-op
+				componentID := livedebugging.ComponentID(c.opts.ID)
```

Comment:
Do we need to create new `ComponentID` each time?

Reply:
good point, moved it out of the for loop